unturned/redist/OpenMod.Unturned.Redist.nuspec

```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
   <metadata>
     <id>OpenMod.Unturned.Redist</id>
-    <version>3.21.11.0-beta2</version>
+    <version>3.21.11.0-beta3</version>
```

Comment:
Shouldn't it be `3.21.13.0-beta1`?

Reply:
I merged by accident but its not important